src/models/artifact.ts

```diff
@@ -0,0 +1,16 @@
+export const artifactTypes = [
+  'result',
+  'markdown',
+  'table',
+  'unknown',
+] as const
+
+export type ArtifactType = typeof artifactTypes[number]
+
+export type Artifact = {
```

Comment:
This isn't a full copy of the `Artifact` type from the downstream ui-library repo, it only includes fields I think we'll need. We do this with state types as well. Open to ideas on a better strategy, but this will keep the train moving for now.

Reply:
Yeah this makes sense to me. We might also want to have an api endpoint we can hit that returns a striped down version of the artifacts. That way we're not sending giant data tables to the browser just to show the title in a box.